CHANGELOG.md

```diff
@@ -4,11 +4,12 @@
 
 - The source code preview for functions that return a type incompatible with
   the functions annotations has been improved to be more precise.
+- A helpful error message is rendered if an enum field contains a generic type
+  that has not been declared.
 
 ## v0.4.1 - 2019-09-29
 
 - Struct types with parameterised fields are now registered with the correct
-  number of type parameters.
```

Comment:
What happened here?  O.o

Reply:
I messed up the rebase it seems. Thanks for alerting me to this!